experimenter/experimenter/experiments/api/v5/types.py

```diff
@@ -754,3 +758,6 @@ def resolve_required_experiments_branches(self, info):
 
     def resolve_subscribers(self, info):
         return self.subscribers.all().order_by("username")
+
+    def resolve_feature_has_live_multifeature_experiments(self, info):
+        return self.feature_has_live_multifeature_experiments
```

Comment:
I think if the field name matches a field right on the model it'll do this automatically?